src/Illuminate/Contracts/Auth/Access/Gate.php

```diff
@@ -74,6 +74,24 @@ public function denies($ability, $arguments = []);
     public function check($ability, $arguments = []);
 
     /**
+     * Determine if any of the given ability should be granted.
+     *
+     * @param  iterable  $abilities
+     * @param  array|mixed  $arguments
```

Comment:
I realize that. Please take a look at the other method annotations in this class, and you will see that it is a consistent pattern.